### PR TITLE
Bump github.com/moby/buildkit from 0.11.6 to 0.12.5

### PR DESCRIPTION
**Pull request by antoinerg**
To address:
- CVE-2024-23652
- CVE-2024-23653

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -40,6 +40,11 @@ require (
 	sigs.k8s.io/yaml v1.4.0
 )
 
+require (
+	github.com/felixge/httpsnoop v1.0.4 // indirect
+	go.opentelemetry.io/otel/metric v1.21.0 // indirect
+)
```
Comment (tomkennedy513) [inline go.mod]: What are these new requires for

**Comment by tomkennedy513** [comment]
You can ignore the failing tests for building images. They are failing since the actions are running in a fork

**Comment by antoinerg** [comment]
> You can ignore the failing tests for building images. They are failing since the actions are running in a fork

That's too bad. I was hoping to trigger the e2e tests to make sure the changes are :heavy_check_mark: 

cc @tomkennedy513

**Comment by tomkennedy513** [comment]
> > You can ignore the failing tests for building images. They are failing since the actions are running in a fork
> 
> 
> 
> That's too bad. I was hoping to trigger the e2e tests to make sure the changes are :heavy_check_mark: 
> 
> 
> 
> cc @tomkennedy513 

Ya we always intending on having a pr flow that would build the images to a local registry and then do e2e but never got around to getting it implemented. I'm ok merging, but I was just curious about those 2 new requires that were added.

**Comment by antoinerg** [comment]
> Ya we always intended on having a pr flow that would build the images to a local registry and then do e2e but never got around to getting it implemented. I'm ok merging, but I was just curious about those 2 new requires that were added.

I removed the block then ran `go mod tidy` again to produce https://github.com/buildpacks-community/kpack/pull/1565/commits/b0d7900970e646526a527812cd46403328a96934. They are new indirect dependencies!

**Review by tomkennedy513 — APPROVED** [review]

**Comment by antoinerg** [comment]
@tomkennedy513 thank you for the fast review. I'm not authorized to merge this pull request but please go ahead and merge it!

Is there a timeline for a new patch release of `kpack` that would include those changes?

Thanks again!

**Comment by tomkennedy513** [comment]
> @tomkennedy513 thank you for the fast review. I'm not authorized to merge this pull request but please go ahead and merge it!
> 
> Is there a timeline for a new patch release of `kpack` that would include those changes?
> 
> Thanks again!

I can to release it when the pipeline finishes. can you also cherry pick these changes to main and open a PR?

**Comment by antoinerg** [comment]
> can you also cherry pick these changes to main and open a PR?

Absolutely! Please see https://github.com/buildpacks-community/kpack/pull/1566

**Comment by antoinerg** [comment]
> I can release it when the pipeline finishes.

Awesome! :rocket:

**Comment by tomkennedy513** [comment]
> > I can release it when the pipeline finishes.
> 
> Awesome! 🚀

0.13.3 is released! thank you for working on this

**Pull request merged**